sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/NonFoldableLiteral.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.types._
  * A literal value that is not foldable. Used in expression codegen testing to test code path
  * that behave differently based on foldable values.
  */
-case class NonFoldableLiteral(value: Any, dataType: DataType) extends LeafExpression {
+case class NonFoldableLiteral(var value: Any, dataType: DataType) extends LeafExpression {
```

Comment:
I thought this should be OK since the literal is non foldable and this class is only in the testing package.